### PR TITLE
[master] Zil 5090 load iso server accounts for quick testing

### PR DESCRIPTION
**Pull request by Steve-White-UK**
## Description
Here are the changes I used to get the node working with isolated server accounts and make testing pass.

## Backward Compatibility

- [ ] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [x] small-scale cloud test

### CODE REVIEW

File: src/libNode/Node.cpp [hunk 1]
```diff
@@ -279,8 +312,15 @@ void Node::Init() {
   const auto& dsBlock = m_mediator.m_dsBlockChain.GetBlock(0);
   m_mediator.m_blocklinkchain.AddBlockLink(0, 0, BlockType::DS,
                                            dsBlock.GetBlockHash());
+#ifndef PRODUCTION_VERSION
```
Comment (yaron-zilliqa) [inline src/libNode/Node.cpp:315]: Is this meant to be PRODUCTION_BUILD?

File: src/libNode/Node.cpp [hunk 2]
```diff
@@ -60,6 +60,39 @@ const unsigned int MIN_CHILD_CLUSTER_SIZE = 2;
 
 #define IP_MAPPING_FILE_NAME "ipMapping.xml"
 
+#ifndef PRODUCTION_BUILD
+namespace {
+// testing version
+[[maybe_unused]] int readAccountJsonFromFile(const string& path) {
```
Comment (yaron-zilliqa) [inline src/libNode/Node.cpp:66]: I suggest using std::filesystem::path for paths.

File: src/libNode/Node.cpp [hunk 3]
```diff
@@ -60,6 +60,39 @@ const unsigned int MIN_CHILD_CLUSTER_SIZE = 2;
...
+  ifstream in(path.c_str());
+
+  if (!in) {
+    cerr << "Cannot open file \n" << path << endl;
+    return -1;
+  }
+  Json::Value _json;
```
Comment (yaron-zilliqa) [inline src/libNode/Node.cpp:73]: Identifier prefixed with _ or double _ are reserved actually.

Comment (Steve-White-UK) [reply src/libNode/Node.cpp:73]: Copied from isolated server code untouched by Yorkshire fingers.

File: src/libNode/Node.cpp (hunk 2 above)
Comment (Steve-White-UK) [reply src/libNode/Node.cpp:66]: Again if we find this useful then we should clean it up, its from our own code base.

File: src/libNode/Node.cpp (hunk 1 above)
Comment (Steve-White-UK) [reply src/libNode/Node.cpp:315]: I put this in as an example to show how to read in the same accounts we use in isolated-server for testing, it means then we have a stable set of results to test against whilst playing with this rig.

If people like this facility then it should be implemented as a command line option, there is some already but with input in other formats, this uses the exact same file as the isolated server.

**Review by JamesHinshelwood — COMMENTED** [review]
IMO we should try to make this work with the genesis accounts, rather than adding code which will only be useful for development and actively harmful in production. We can compile it out in production builds, but it still adds unnecessary complexity in my view.

**Comment by Steve-White-UK** [comment]
Closed as not relavant now

**Pull request closed without merging**